dev/util.h

```diff
@@ -14,5 +15,13 @@ namespace sqlite_orm {
                                         sqlite3_errmsg(db));
             }
         }
+
+        static void perform_void_exec(sqlite3 *db, const std::string &query) {
+            int rc = sqlite3_exec(db, query.c_str(), NULL, NULL, NULL);
```

Comment:
please use `nullptr` instead of `NULL`. `NULL` is legacy C macro and `nullptr` is strong typed

Reply:
Done.